plugins/lookup/tss.py

```diff
@@ -26,6 +26,11 @@
         description: The integer ID of the secret.
         required: true
         type: int
+    secret_path:
+        description: Indicate a full path of secret including folder and secret name when the secret ID is set to 0.
```

Comment:
The UX of this feels strange. Using a magic value `0` to specify the real value somewhere else seems very confusing.

Reply:
You removed that part from the description, but in the example and the code the magic value is still there.

Reply:
Actually, Get secret API of Delinea secret server required input in this format when fetching secret by path.

Reply:
Then why does the documentation not say this? (But only the example and the code itself?)

Reply:
@felixfontein added in documentation also